.releaserc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "branches": ["main", "dev"],
```

Comment:
for _prerelease_ you need to tell it. It's probably easier to go with the default settings for alpha/beta branches. https://semantic-release.gitbook.io/semantic-release/recipes/release-workflow/pre-releases

```suggestion
```